flexget/plugins/clients/aria2.py

```diff
@@ -61,12 +61,12 @@ def _get_req_params(self, method, params=None):
             del req_params['params']
         return req_params
 
-    def _default_error_handle(code, message):
-        logger.critical('Fault code {} message {}', code, message)
-        raise plugin.PluginError(f'Fault code {code} message {message}', logger)
+    def _default_error_handle(self, message):
+        logger.critical('Fault code {} message {}', self, message)
+        raise plugin.PluginError(f'Fault code {self} message {message}', logger)
```

Comment:
I think in this one it's secretly being used as a `@staticmethod` by virtue of referencing these functions from the function signatures before the class actually gets to make them into methods. I believe it won't break anything to just put `@staticmethod` on the two methods in this file and leave the variable names the same.